player/db/rounds/rounds.go

```diff
@@ -1 +1,42 @@
 package roundsdb
+
+import (
+	"context"
+	"database/sql"
+	"github.com/adamhicks/ctrlaltdefeat/player"
+)
+
+func InsertRoundParts(ctx context.Context, dbc *sql.DB, matchId, roundId int, playerId string, rank, p1Part, p2Part, p3Part, p4Part int) (int64, error) {
```

Comment:
the rank and parts will be int64, not needed for data range but will avoid a lot of casting.